modules/todo.js

```diff
@@ -0,0 +1,97 @@
+import { parseInt } from 'lodash';
+import Selector from './selectors.js';
+import checkStatus from './statusCheck.js';
+
+const ObjectSelec = new Selector();
+export default class Todo {
+  constructor() {
+    this.todos = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  visable = () => {
+    if (this.todos.length === 0) {
+      ObjectSelec.clear.classList.add('display');
+    } else {
+      ObjectSelec.clear.classList.remove('display');
+    }
+  }
+
+  dispaly = () => {
+    ObjectSelec.todoBody.querySelector('.js-todo-list').innerHTML = '';
+    this.todos.forEach((todo) => {
+      const todoList = `<li data-id="${todo.id}" data-status="${todo.status}">
+      <label for="${todo.id}">
+      <input type="checkbox" class="check" id="${todo.id}" value="${todo.id}"
+      ${todo.status === 'completed'}/>
+      <input type="text" class="js-edit-input" value="${todo.task}"/>
+      </label>
+          <div class="action">
+          <button class="js-delete">
+              <i class="ri-delete-bin-fill"></i>
+          </button>
+          </div> 
+      </li>`;
+      ObjectSelec.todoBody.querySelector('.js-todo-list').innerHTML += todoList;
+    });
+    this.visable();
+  }
+
+  addTask = (e) => {
+    e.preventDefault();
+    const todo = ObjectSelec.input.value;
+    if (!todo) return;
+    if (this.todos.length === 0) {
+      const myTodo = { id: 1, task: todo, status: false };
+      this.todos.push(myTodo);
+    } else {
+      const index = this.todos.length + 1;
+      const myTodo = { id: index, task: todo, status: false };
+      this.todos.push(myTodo);
```

Comment:
Kindlly I suggest you to create a new class Task that have 3 attributes id, description and status. instead of using literal objects you can work easily a class of object.